starknet_devnet/blueprints/base.py

```diff
@@ -44,20 +48,20 @@ def extract_positive(request_json, prop_name: str):
     return value
 
 
-def extract_hex_string(request_json, prop_name: str) -> int:
-    """Parse value from hex string to int"""
+def hex_converter(request_json, prop_name: str, convert=custom_int) -> int:
```

Comment:
Well `custom_int(x)` isn't the same as `int(x, 16)`. The former assumes a decimal string if it doesn't start with `0x`.

Reply:
I just wanted to use the existing function, should I create a new one with `int(x, 16)`?

Reply:
ok the conclusion here was to stick with `custom_int`, and later will address this together with https://github.com/Shard-Labs/starknet-devnet/issues/368